src/util/types/interval.ts

```diff
@@ -13,7 +17,11 @@ export const intervalLabels: Record<Interval, string> = {
   I1M: "1m",
   I5M: "5m",
   I15M: "15m",
+  I30M: "30m",
   I1H: "1H",
+  I4H: "4H",
   I6H: "6H",
+  I12H: "12H",
   I1D: "1D",
+  I7D: "7D",
```

Comment:
Any reason why minutes are in lower case `m` and days / hours upper `H, D` ?

Reply:
Likely to differentiate between minutes and months. Trading view for example will omit 'm' complete and a bare integer indicates minutes. So probably best to just follow the convention set in pennant here.